utils/lib/feedback.ex

```diff
@@ -398,8 +398,10 @@ defmodule Utils.Feedback do
 
     now = DateTime.utc_now()
     assert tomorrow.year == now.year
-    assert tomorrow.month == now.month
-    assert tomorrow.day == now.day + 1
+
+    assert (tomorrow.month == now.month && tomorrow.day == now.day + 1) ||
```

Comment:
We want to move away from using Utils.feedback in favor of either doc testing or providing hints (see CONTRIBUTING.md)

I'm still debating the best way to test for exercises, but so far am leaning towards doctests (which will automatically run in Livebook soon)

Reply:
I'm noticing you did use a hint instead of writing a test. Can you explain the code above to me?